src/index.ts

```
@@ -547,6 +547,11 @@ export class Bigtable {
     options: InstanceOptions,
     callback?: CreateInstanceCallback
   ): void | Promise<CreateInstanceResponse> {
+    if (!options.clusters) {
```

Since we require an object, let's throw if that's missing, too. Otherwise, this would be a TypeError ("Cannot read property 'clusters' of undefined")

---

Great point!
Done!